src/langcheck/metrics/_model_management.py

```diff
@@ -0,0 +1,216 @@
+                    tokenizer, model = load_auto_model_for_text_classification(
+                        model_name=model_name,
+                        tokenizer_name=tokenizer_name,
+                        revision=revision)
+                    return tokenizer, model
+                else:
+                    raise KeyError(f'Loader {loader_type} not supported yet.')
+            else:
+                raise KeyError(f'Metric {metric} not supported yet.')
+        else:
+            raise KeyError(f'language {language} not supported yet')
+
+    def list_current_model_in_use(self, language='all', metric='all'):
+        '''
+        List the models currently in use.
+
+        Args:
+            language: The abbrevation name of language
+            metric: The evaluation metric name
+        '''
+        df = pd.DataFrame.from_records(
+            [(lang, metric_name, key, value)
+             for lang, lang_model_settings in self.config.items()
+             for metric_name, model_settings in lang_model_settings.items()
+             for key, value in model_settings.items()],
+            columns=['language', 'metric_name', 'attribute', 'value'])
+
+        # The code below would generate a dataframe:
+        # |index| language | metric_name | loader | model_name | revision |
+        # |.....|..........|.............|........|............|..........|
+        df_pivot = df.pivot_table(index=['language', 'metric_name'],
+                                  columns="attribute",
+                                  values="value",
+                                  aggfunc='first').reset_index().drop(
+                                      columns=["attribute"]).reset_index()
+        df_pivot.columns = [
+            'language', 'metric_name', 'loader', 'model_name', 'revision'
+        ]
+
+        if language == 'all' and metric == 'all':
+            pprint(df_pivot)
+        else:
+            if language != "all":
+                df_pivot = df_pivot.loc[df_pivot.language == language]
+            if metric != 'all':
+                df_pivot = df_pivot.loc[df_pivot.metric_name == metric]
+            pprint(df_pivot)
+
+    def validate_config(self, language='all', metric='all'):
+        '''
+        Validate configuration.
+
+        Args:
+            language: The name of the language. Defaults to 'all'.
+            metric: The name of the metric. Defaults to 'all'.
+        '''
+
+        def check_model_availability(model_name, revision):
+            if revision is None:
+                url = f"https://huggingface.co/api/models/{model_name}"
+            else:
+                url = f"https://huggingface.co/api/models/{model_name}/revision/{revision}"  # NOQA:E501
+            response = requests.get(url)
+            return response.status_code == 200
+
+        config = deepcopy(self.config)
+        for lang, lang_setting in config.items():
+            if language == 'all' or lang == language:
```

Comment:
```suggestion
            if language != 'all' and lang != language:
                continue
```
so that we can remove one level of nesting